script/bootstrap

```diff
@@ -15,3 +15,6 @@ if [ -f .env.${NODE_ENV} ]; then
 fi
 
 npm install
+
+# Make sure mocha and coffee are in the PATH
+export PATH="node_modules/.bin:$PATH"
```

Comment:
It looks like `make test` is called by [cibuild](https://github.com/afeld/hubot/blob/testing/script/cibuild), so I just updated the Makefile to call `script/test` instead.